ci-operator/jobs/openshift/elasticsearch-operator/openshift-elasticsearch-operator-feature-es6x-presubmits-override.yaml

```diff
@@ -0,0 +1,171 @@
+presubmits:
+  openshift/elasticsearch-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - feature-es6x
+    context: ci/prow/e2e-operator
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-elasticsearch-operator-feature-es6x-e2e-operator
+    rerun_command: /test e2e-operator
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --branch=feature-es6x
+        - --give-pr-author-access-to-namespace=true
+        - --lease-server=http://boskos
+        - --org=openshift
+        - --repo=elasticsearch-operator
+        - --resolver-address=http://ci-operator-configresolver-ci.svc.ci.openshift.org
+        - --secret-dir=/usr/local/e2e-operator-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-operator
+        - --template=/usr/local/e2e-operator
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_VARIANT
+          value: xlarger_worker
```

Comment:
Isn't `CLUSTER_VARIANT` used to set up a size of the master nodes already? This will need to follow a modification in the template that allows the node size to be parametrized, and I guess the name of the parameter will probably be different.

Reply:
> Isn't `CLUSTER_VARIANT` used to set up a size of the master nodes already? This will need to follow a modification in the template that allows the node size to be parametrized, and I guess the name of the parameter will probably be different.

It is.  @bparees suggest to reuse it for now